src/utils.tsx

```diff
@@ -288,17 +288,21 @@ export const sendPerfTiming = ({
   _tracking = tracking,
   actualDuration,
   id,
-  phase,
+  url,
 }: {
   _tracking?: typeof tracking;
   actualDuration: number;
   id: string;
-  phase: string;
```

Comment:
what was `phase` and why don't we need it any longer?

Reply:
`phase` is passed into the `onRender` callback, and it represents the render phase, which is either `mount` or `unmount`. I was passing it to GA as an extra piece of data, but we really don't care about the difference between `mount` and `unmount` for these timings, and we only have room to send so much data to GA, so I have opted to send the `url` instead of the `phase`.